### ISSUE TITLE
Turning on/off for users

### ISSUE THREAD

**Issue by dave-ryan** [iiiklox/niconico-twitch#1]
Hello! I tried making an overlay myself but that ran into the ground under the TOS of Twitch. Anyways, I was wondering if if you use this during your stream, will it show up for anyone? and can they turn it off somehow? @iiiklox

**Comment by tekigg** [comment]
Hello!

Are you referring to [Twitch stream extensions](https://dev.twitch.tv/extensions)? In which case no, unfortunately. This is a client-side overlay so what you see is what you get, no toggles for viewers. 

Although, I did some digging and I found [this Twitch extension](https://dashboard.twitch.tv/extensions/gh70y1spw727ohtgzbhc0hppvq9br2-0.2.0). This is a viewer-sided chat overlay that seems to have the functionality you’re looking for. I haven’t used it before, though!

**Comment by dave-ryan** [comment]
Yeah unfortunately that is broken right now, so I went on a journey to remake the extension from scratch, knowing nothing about twitch extensions. I ended up recreating it successfully... only to find out that it breaks TOS apparently. I'd need permission from every user before I can 'publish' what they typed, show their username, as well as have the broadcaster have the ability to moderate **every single message** before displaying it. Your project looks really cool though!
